chirps_utility.py

```diff
@@ -0,0 +1,83 @@
+from chirps import *
+from csv_utility import *
+from user_utility import *
+from random_id_generator import *
+
+class ChirpsUtility:
+
+	def view_chirps(chirps_dict, current_user):
+		"""
+		Prints out all public and private chirps that the current user is a part of
+
+		Arg- chirps dictionary (from CSV file), currently selected user
+		"""
+		counter = 1
+		if current_user != None:
+			print("")
+			print(" ~ PRIVATE CHIRPS ~ ")
+			for k, v in chirps_dict.items():
+				if v[1] == current_user.screen_name:
+					print("{0}. {1}: {2}".format(counter, v[0], v[2]))
+					counter += 1
+			print("")
+			ChirpsUtility.print_public_chirps(chirps_dict, counter)
+		else:
+			ChirpsUtility.print_public_chirps(chirps_dict, counter)
+
+	def print_public_chirps(chirps_dict, counter):
```

Comment:
Nice idea passing the counter along to maintain menu ordering across listings of chirp types.